corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -21,6 +21,7 @@
 
 
 def delete_sync_logs(before_date, limit=1000, num_tries=10):
+    # Todo: convert to SQL including get_synclog_ids_before_date
```

Comment:
We don't need to fully delete them but we should blank out the `doc` field to reclaim some space.

Reply:
One benefit of not deleting them is that the synclog report would work again. Perhaps we can compromise and clear the json data after 30 days (or on the next sync) but keep the metadata for a longer period.

Reply:
Yeah  that makes sense. Though will that be fixed in the data warehouse? I recall seeing something about a synclogfact table.

Reply:
I think we can let this do couch deletion (instead of SQL) till we have around 30 days of SQL synclogs, so that this will continue to be useful for deleting couch docs. I will convert this to SQL version in a separate PR. Does that sound right?

Reply:
Seems OK I guess.